buildSrc/src/main/kotlin/otel.errorprone-conventions.gradle.kts

```diff
@@ -21,6 +26,9 @@ tasks {
         disableWarningsInGeneratedCode.set(true)
         allDisabledChecksAsWarnings.set(true)
 
+        // Ignore warnings for generated and vendored classes
+        excludedPaths.set(".*/build/generated/.*|.*/internal/shaded/.*")
```

Comment:
do we really want to exclude our own `internal` classes from our errorprone checks?

Reply:
oh..it's just `internal/shaded`. Never mind!